index.html

```diff
@@ -93,3 +93,450 @@ <h2 class="subSentenceText">I'm a Software Developer</h2>
+              </ul>
+            </ul>
+
+            <ul class="tech">
+              <li class="Language1">
+                <img src="Rectangle3.png" alt="Rectangle" /> skills
+              </li>
+              <ul class="TechStacks">
+                <li>Database Management</li>
+                <li>Version Control</li>
+                <li>CLI</li>
+                <li>API Design</li>
+                <li>Web Developement</li>
+              </ul>
+            </ul>
+          </ul>
+        </div>
+      </div>
+    </section>
+
+    <!-- html form -->
+
+    <section class="Contact" id="Contact">
+      <div class="form">
+        <a href="#form"></a>
+        <div class="mainContent">
+          <h2 class="smallPara">
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h2>
+          <form
+            class="firstForm"
+            id="form"
+            method="post"
+            action="https://formspree.io/f/xgedqoye"
+          >
+
+          
+            <ul class="input">
+              <div class="subInput">
+              <li class="newInputName">
+                <input
+                  name="Name"
+                  type="Text"
+                  id="Name"
+                  placeholder="Full name"
+                  maxlength="30"
+                  required
+                />
+                <div class="gaps1"></div>
+              </li>
+
+              <li class="newInputName01">
+                <input
+                  name="Name"
+                  type="Text"
+                  id="Name"
+                  placeholder="First name"
+                  maxlength="30"
+                  required
+                />
+                <div class="gaps1"></div>
+              </li>
+              <li class="newInputName02">
+                <input
+                  name="LastName"
+                  type="Text"
+                  id="Name"
+                  placeholder="Last name"
+                  maxlength="30"
+                  required
+                />
+                <div class="gaps11"></div>
+              </li>
+
+              <li class="newInputEmail">
+                <input
+                  name="Email"
+                  type="Email"
+                  id="Email"
+                  placeholder="Email address"
+                  maxlength="30"
+                  required
+                />
+                <div class="gaps2"></div>
+              </li>
+
+              <li class="textArea" id="message">
+                <textarea
+                  class="textArea1"
+                  name="message"
+                  id="textArea1"
+                  placeholder="Write me something..."
+                  maxlength="500"
+                  required
+                ></textarea>
+              </li>
+
+              <li>
+                <button class="submit" type="submit" value="submit">
```

Comment:
> Implement the button interactions (enable, hover, etc.).

- [ ] The button interactions like the hover is not implemented in this submit button as stated in the project requirement. I'll suggest you look at the Figma designs dn get the text and background color for different button states. :wave: 

- - FIGMA:
![Screenshot from 2023-07-11 13-50-12](https://github.com/chiedozie013/portfolio-microverse/assets/107632780/d01f64d5-f311-452d-ba84-6d51ed13585f)
- [ ] When I try to submit a form, I get this error in the console. I'll suggest you look into this and fix it. And also try and make sure the form submits from your end to `form-spree` before request ing the next review. :smile: 

- - Console error screenshot:
![Screenshot from 2023-07-12 16-01-40](https://github.com/bhushan354/My-Portfolio/assets/107632780/335fca03-2b06-488b-b102-b7d07c163fa2)

  
- [ ] Also I'll suggest you adjust the width of this button to fully match the Figma design as your button width is a bit smaller than the given design: :+1: :+1: